classes/sota.bbclass

```diff
@@ -51,6 +51,8 @@ GARAGE_SIGN_KEYNAME ?= "garage-key"
 GARAGE_TARGET_NAME ?= "${OSTREE_BRANCHNAME}"
 GARAGE_TARGET_VERSION ?= ""
 GARAGE_TARGET_URL ?= ""
+GARAGE_TARGET_EXPIRES ?= ""
+GARAGE_TARGET_EXPIRE_AFTER ?= ""
```

Comment:
What if none of these are set?

Reply:
Then nothing is specified and the default from `garage-sign` is applied.

Reply:
but `garage-sign` doesn't have a default anymore. By default it will use what is in the unsigned role. If the user just uses bitbake it will expire.

Reply:
Default still appears to be one month.

Reply:
The default of `targets init` is 31 days, yes. This doesn't get increased when signing anymore, though.

Reply:
Ah, sorry, missed a comment. So what's the right thing to do? Should we set a default of our own (31 days, presumably) if both variables are unset?

Reply:
Depends on what behavior you want. If you want the default behavior to mimic the previous behavior, you could use `GARAGE_TARGET_EXPIRE_AFTER ?= "1m"` if I understand that syntax correctly, `1m` standing for "1 month". (Actually before it was more like `GARAGE_TARGET_EXPIRE_AFTER ?= "31d"`, but `1m` probably makes more sense in general.)

Reply:
Thanks, done!